spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RuntimeAppsController.java

```diff
@@ -169,10 +171,18 @@ private void enrichWithMetrics(List<AppStatus> statuses) {
 				String trackingKey = appInstanceStatus.getAttributes().get("guid");
 				if (metricsInstanceMap.containsKey(trackingKey)) {
 					ApplicationsMetrics.Instance metricsAppInstance = metricsInstanceMap.get(trackingKey);
-					appInstanceStatus.getAttributes().put("metrics.integration.channel.input.receiveRate",
-							String.format("%.2f", metricsAppInstance.getIncomingRate()));
-					appInstanceStatus.getAttributes().put("metrics.integration.channel.output.sendRate",
-							String.format("%.2f", metricsAppInstance.getOutgoingRate()));
+					List<Metrics> metrics = metricsAppInstance.getMetrics();
+					if (metrics != null) {
+						for (Metrics m : metrics) {
+							if (ObjectUtils.nullSafeEquals("integration.channel.input.send.mean", m.getName())) {
+								appInstanceStatus.getAttributes().put("metrics.integration.channel.input.receiveRate",
+										String.format("%.2f", m.getValue()));
```

Comment:
maybe we can add `msgs/sec` here to provide units.  We have `%` added in the case of cloud foundry for the mem/cpu/disk utilization.